node_registry.json

```diff
@@ -32,7 +32,7 @@
     "version": "1.27.0",
     "services": [
       {
-        "$schema": "https://raw.githubusercontent.com/DACCS-Climate/DACCS-node-registry/main/node_registry.schema.json#service",
+        "$schema": "https://raw.githubusercontent.com/DACCS-Climate/Marble-node-registry/main/node_registry.schema.json#service",
```

Comment:
All of these changes need to be made in the birdhouse-deploy repo, not here

Reply:
Oh that is interesting. So we would need to make a simultaneous change to birdhouse-deploy as well?

Reply:
Yeah. We'd have to update every `service-config.json.template` file

For example here:

https://github.com/bird-house/birdhouse-deploy/blob/1.29.0/birdhouse/config/finch/service-config.json.template#L2

Reply:
Commit removed.